doc/sphinx-guides/source/developers/sql-upgrade-scripts.rst

```diff
@@ -2,40 +2,38 @@
----------------------------------------------------------------------
+SQL naming conventions
+----------------------
+In order to make the migrations work you must follow the naming conventions of the flyway - https://flywaydb.org/documentation/migrations#sql-based-migrations
 
-If you are creating a new database table (which maps to an ``@Entity`` in JPA), you do not need to create or update a SQL upgrade script. The reason for this is that we use ``create-tables`` in ``src/main/resources/META-INF/persistence.xml`` so that new tables are automatically created by Glassfish when you deploy your war file.
+How to Determine if You Need to Create SQL
+------------------------------------------
 
-If you are doing anything other than creating a new database table such as adding a column to an existing table, you must create or update a SQL upgrade script.
+If you are creating a new database table (which maps to an ``@Entity`` in JPA), you do not need to create. The reason for this is that we use ``create-tables`` in ``src/main/resources/META-INF/persistence.xml`` so that new tables are automatically created by Glassfish when you deploy your war file.
 
-How to Create or Update a SQL Upgrade Script
---------------------------------------------
+If you are doing anything other than creating a new database table such as adding a column to an existing table, you must create SQL.
 
-We assume you have already read the :doc:`version-control` section and have been keeping your feature branch up to date with the "develop" branch.
-
-First, check https://github.com/IQSS/dataverse/tree/develop/scripts/database/upgrades to see if a SQL upgrade script for the next release already exists. For example, if the current release is 4.9.4 and the next release will be 4.10, the script will be named ``upgrade_v4.9.4_to_v4.10.sql``. If the script exists, just add your changes to the bottom of it.
+How to Create SQL
+-----------------
 
-If no SQL upgrade script exists, look at https://github.com/IQSS/dataverse/milestones to figure out the name of the next milestone and create a script using the naming convention above.
-
-As with any task related to Dataverse development, if you need any help writing SQL upgrade scripts, please reach out using any of the channels mentioned under "Getting Help" in the :doc:`intro` section.
+We assume you have already read the :doc:`version-control` section and have been keeping your feature branch up to date with the "develop" branch.
 
-Please note that we are aware of the problem of merge conflicts in the SQL update script as well as how the next version number can change at any time. Please see the :doc:`making-releases` section for how we are running an experiment having to do with release notes that might help inform an improvement of our process for developing SQL upgrade scripts.
+First, check https://github.com/IQSS/dataverse/tree/develop/src/main/resources/db/migration to see if a SQL numbers are not colliding with your's.
```

Comment:
IMHO we could opt for versions based on `VYYYYMMDDIDX` instead to avoid this for the majority of cases?.

Reply:
I think it should be responsibility of the reviewer, there could be even more mess with this system, E.g. user makes PR in 2019.02.15 with same sql name (with id 1) it was merged in 2019.02.20, now user makes pr in 2019.02.16 (with id 1) it was merged in 2019.02.19. Now flyway will crash because it was out of order and not only you need to track id you also need to track dates now.